docs/data/charts/zoom-and-pan/ZoomSlider.tsx

```diff
@@ -1,7 +1,7 @@
 import { ScatterChartPro } from '@mui/x-charts-pro/ScatterChartPro';
 import * as React from 'react';
 
-const dataLength = 800;
+const dataLength = 801;
```

Comment:
Increased to 801 otherwise the last value in the zoom slider tooltip would be 799 which isn't a round number and it made me feel uneasy 😛